selfdrive/car/tests/test_models_segs.txt

```diff
@@ -0,0 +1,3888 @@
+# HONDA ODYSSEY 2018
```

Comment:
this file should probably be in LFS (or even pushed to the azure CI bucket)

Reply:
was going to add to LFS. adding list of private routes to public Azure bucket doesn't make sense to me